### ISSUE TITLE
creating a new astro project crashes

### ISSUE THREAD

**Issue by lifers** [oven-sh/bun#11824]
### How can we reproduce the crash?

run `bun create astro`

### Relevant log output

```shell
astro   Launch sequence initiated.

   dir   Where should we create your new project?
         ./lifers-blog-astro

  tmpl   How would you like to start your new project?
         Include sample files

    ts   Do you plan to write TypeScript?
         Yes

   use   How strict should TypeScript be?
         — Strict
         > Strictest
         — Relaxed ============================================================
Bun v1.1.13 (bd6a6051) Windows x64
Args: "bun" "C:\Users\SF-NAG~1\AppData\Local\Temp\bunx-1692489392-create-astro@latest\node_modules\create-astro\create-astro.mjs"
Features: jsc Bun.stdin(2) Bun.stdout fetch spawn(2)
Builtins: "bun:main" "node:assert" "node:buffer" "node:child_process" "node:crypto" "node:dns/promises" "node:events" "node:fs" "node:fs/promises" "node:https" "node:os" "node:path" "node:process" "node:readline" "node:stream" "node:stream/consumers" "node:string_decoder" "node:tty" "node:util" "node:util/types" "node:zlib"
Elapsed: 24554ms | User: 93ms | Sys: 31ms
RSS: 0.20GB | Peak: 0.20GB | Commit: 0.28GB | Faults: 51846

panic(main thread): Segmentation fault at address 0x5
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:
```

### Stack Trace (bun.report)

Bun v1.1.13 ([`bd6a605`](<https://github.com/oven-sh/bun/tree/bd6a6051203ff577240615ff4e242270035a36b7>)) on windows x86_64 [AutoCommand]

Segmentation fault at address 0x00000005

- *5 unknown/js code*
- `??`
- *4 unknown/js code*

**Comment by nektro** [comment]
looks like this was fixed. if this still reproduces for you on the latest version of Bun, please let us know and we'll reopen but i think for now we'll close. (you may need to run `bun upgrade`)

![Image](https://github.com/user-attachments/assets/21fa3023-8f5a-428e-ad43-6cd1d9ba3796)

**Comment by nektro** [comment]
oops, looks like i forgot the `--bun` flag. using that i get:

![Image](https://github.com/user-attachments/assets/b99c2dd3-c9d2-426f-bb8f-a8d564fa1645)

**Comment by heimskr** [comment]
Fixed by #15357.
